src/main/kotlin/no/nav/personbruker/dittnav/api/common/InnloggetBruker.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.personbruker.dittnav.api.common
+
+import no.nav.security.token.support.core.jwt.JwtToken
+
+class InnloggetBruker(val token: JwtToken) {
+
+    fun getBearerToken(): String {
+        return "Bearer " + token.tokenAsString
+    }
+
+    fun getIdentFromToken(): String {
+        val ident = token.jwtTokenClaims.getStringClaim("pid")
```

Comment:
Kunne default vært å hente ut ident fra sub, jeg tror det er det vi får fra Azure. Så kan man verifisere at identen er 11 siffer, hvis den ikke er det så henter man pid. Da er vi litt tryggere hvis det skulle begynne å være noe som ligger i pid-claim-et fra Azure. Man trenger da også bare å hente ut et claim en gang, i stede for å alltid gjøre det to ganger i dev og prod.